src/Controller/CRUDController.php

```diff
@@ -627,6 +627,7 @@ public function createAction()
                         return $this->renderJson([
                             'result' => 'ok',
                             'objectId' => $this->admin->getNormalizedIdentifier($newObject),
+                            'objectName' => $this->escapeHtml($this->admin->toString($newObject)),
```

Comment:
out of curiosity: why html escape the name here?

Reply:
I get this line from the edit action some line above (line 363)
I think it's for XSS purpose.